### PR TITLE
[Feature][Branch][Rating] Apply Knobs addon on Storybook

### PR DESCRIPTION
**Pull request by devethan**
## Description

I try to apply Knobs addon for Rating component.

## Related Issues

#53 

## Tests

No test. I just updated storybook file.

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui-native/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: storybook/stories/Rating.stories.tsx [hunk 1]
```diff
@@ -19,53 +22,31 @@ const Container = styled.View`
...
-  width: 100%;
-  height: 1px;
-  margin: 15px 0;
-  background-color: lightgrey;
-`;
 
 function Default(): React.ReactElement {
-  const [stars1, setstars1] = React.useState(3);
-  const [stars2, setstars2] = React.useState(3);
+  const [value, setvalue] = React.useState(number('value', 3));
```
Comment (JeffGuKang) [inline storybook/stories/Rating.stories.tsx]: Normally, we use camelcase as `[value, setValue]`. If you use autocomplete in VSCode, try `tab` after write `value`. 
For example, 

1. type `usesta` and choose autocomplete and enter.
2. type `value` for your state value.
3. tab to input initial value.

**Review by JeffGuKang — CHANGES_REQUESTED** [review]

File: storybook/stories/Rating.stories.tsx [hunk 2]
```diff
@@ -19,53 +22,31 @@ const Container = styled.View`
...
-      <Rating total={5} value={stars1} disabled />
-      <Divider />
-      <Title>10 Stars</Title>
-      <Rating total={10} value={stars2} onChange={handleChange2} />
-      <Result>Selected: {stars2} stars</Result>
-      <Divider />
+      <Rating
+        total={number('total', 5)}
+        value={value}
+        onChange={!disabled && handleChange}
```
Comment (JeffGuKang) [inline storybook/stories/Rating.stories.tsx]: If you set `disabled` true, it should to block user response in `Rating` component. Thus, `onChange={handleChange}` looks better.

File: storybook/stories/Rating.stories.tsx (hunk 1 above)
Comment (JeffGuKang) [inline storybook/stories/Rating.stories.tsx]: `number('initValue', 3)` will be better. Cause it looks using for initial value.

**Review by JeffGuKang — CHANGES_REQUESTED** [review]

File: storybook/stories/Rating.stories.tsx (hunk 1 above)
Comment (devethan) [reply storybook/stories/Rating.stories.tsx]: Thanks for your opinion. I think that's better 👍

**Review by JeffGuKang — APPROVED** [review]
LGTM

**Pull request merged**